tetris.sh

```diff
@@ -17,13 +17,60 @@
 #
 # Author: Kirill Timofeev <kt97679@gmail.com>
 #
+# Localized Support: Rojen Zaman <rojen@riseup.net> | lang/README.md
+#
 # This program is free software. It comes without any warranty, to the extent
 # permitted by applicable law. You can redistribute it and/or modify it under
 # the terms of the Do What The Fuck You Want To Public License, Version 2, as
 # published by Sam Hocevar. See http://www.wtfpl.net/ for more details.
 
 set -u # non initialized variable is an error
 
+# show help page when given -h argument
+function help_page() {
+cat <<EOF
+usage: ./`basename $0` -l <language>
+supported languages:
+english (it is defualt, option not required)
+$(ls -I README.md -1t lang/ | cut -d'.' -f1)    
+EOF
+exit 0;
+}
+
+# BEGIN OF LANGUAGE LINES
+# if you want see or add other languages, visit lang/ dir and read lang/README.md
+# default localized values english:
+local_LinesComplated="Lines completed: ";
+local_Level="Level:           ";
+local_Score="Score:           ";
+local_UseCursorKeys="  Use cursor keys";
+local_Or="       or";
+local_Rotate="    s: rotate";
+local_LeftRight="a: left,  d: right";
+local_Drop="    space: drop";
+local_Quit="      q: quit";
+local_ToggleColor="  c: toggle color";
+local_ToggleShowNext="n: toggle show next";
+local_ToggleThisHelp="h: toggle this help";
+local_GameOver="Game over!";
+
+while getopts ":l:h" opt; do
+  case ${opt} in
+    l )
+        langChocie=${OPTARG};
+        . lang/$langChocie.sh &> /dev/null
```

Comment:
We have 2 options here:
1. leave current behavior. This means that in case user asked for non existing language no error will be shown and default english will be used.
2. error message will be shown and program will stop
What do you think about those options?

Reply:
you're right, 
 I will add